src/current/v22.2/changefeed-messages.md

```diff
@@ -177,10 +220,10 @@ By default, [protected timestamps](architecture/storage-layer.html#protected-tim
 
 Protected timestamps will protect changefeed data from garbage collection in the following scenarios:
 
-- The downstream [changefeed sink](changefeed-sinks.html) is unavailable. Protected timestamps will protect changes until you either [cancel](cancel-job.html) the changefeed or the sink becomes available once again. 
+- The downstream [changefeed sink](changefeed-sinks.html) is unavailable. Protected timestamps will protect changes until you either [cancel](cancel-job.html) the changefeed or the sink becomes available once again.
 - You [pause](pause-job.html) a changefeed with the [`protect_data_from_gc_on_pause`](create-changefeed.html#protect-pause) option enabled. Protected timestamps will protect changes until you [resume](resume-job.html) the changefeed.
```

Comment:
```suggestion
- A changefeed with the [`protect_data_from_gc_on_pause`](create-changefeed.html#protect-pause) option enabled is [paused](pause-job.html). Protected timestamps will protect changes until you [resume](resume-job.html) the changefeed.
```

changing format here to read the same as the previous bullet

Reply:
I understand the intent here to parallelize the bullet points, but I think the introduction of the passive voice (i.e. removing the subject "You") obscures the meaning from the reader. 

However, this remind me that an extra piece of information was missing from there, so thank you!